geode-cq/src/distributedTest/java/org/apache/geode/internal/cache/PutAllClientServerDistributedTest.java

```diff
@@ -2541,10 +2474,10 @@ public void testEventIdOutOfOrderInPartitionRegionSingleHop() {
     // client1 add listener and putAll
     client1.invoke(() -> {
       Region<String, TickerData> region = getClientCache().getRegion(regionName);
-      doPutAll(region, keyPrefix, ONE_HUNDRED); // fails in GEODE-7812
+      doPutAll(region, keyPrefix, ONE_HUNDRED);
     });
 
-    client1.invoke(() -> await()
+    client1.invoke(() -> await() // TODO: GEODE-9242 fails on this assertion even with await
         .untilAsserted(() -> assertThat(clientCounter.getCreates()).isEqualTo(ONE_HUNDRED)));
```

Comment:
TODO: Unfortunately, this assertion in testEventIdOutOfOrderInPartitionRegionSingleHop is still flaky